vsts_ci/darwin/continuous-build-darwin.yml

```diff
@@ -1,8 +1,8 @@
 steps:
   - task: UsePythonVersion@0
-    displayName: "Use Python 3.6"
+    displayName: "Use Python $(PYTHON_VERSION_SPEC)"
```

Comment:
What's the value of `$(PYTHON_VERSION_SPEC)`?
```

Reply:
To specify the python to run to avoid failed case which is triggered when py27 and py36 run in the same time.